sanic/request.py

```diff
@@ -212,9 +215,24 @@ def args(self):
         return self.parsed_args
 
     @property
-    def raw_args(self):
+    def raw_args(self) -> dict:
+        if self.app.debug:  # pragma: no cover
+            warnings.simplefilter("default")
+        warnings.warn(
+            "Use of raw_args will be deprecated in "
+            "the future versions. Please use args or not_grouped_args "
+            "properties instead",
+            DeprecationWarning,
+        )
         return {k: v[0] for k, v in self.args.items()}
 
+    @property
+    def not_grouped_args(self) -> list:
+        if self.parsed_not_grouped_args is None:
+            if self.query_string:
+                self.parsed_not_grouped_args = parse_qsl(self.query_string)
```

Comment:
Can we make use of the  `keep_blank_values` so that the no value query string keys are retained ?

Reply:
Or another way to handle it without changing default behavior:
```python
    def get_args(self, keep_blank_values=False, strict_parsing=False, encoding='utf-8', errors='replace'):
        if self.parsed_args is None:
            if self.query_string:
                self.parsed_args = RequestParameters(
                    parse_qs(self.query_string, keep_blank_values, strict_parsing, encoding, errors)
                )
            else:
                self.parsed_args = RequestParameters()
        return self.parsed_args

    args = property(get_args)
```
The same for `query_args`.

Reply:
If someone need to change some default parameters - just call `get_args` method directly.